src/core/varlink-kill.c

```diff
@@ -0,0 +1,30 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "sd-json.h"
+
+#include "kill.h"
+#include "signal-util.h"
+#include "varlink-kill.h"
+#include "varlink-util.h"
+
+int unit_kill_context_build_json(sd_json_variant **ret, const char *name, void *userdata) {
+        KillContext *c = userdata;
+
+        assert(ret);
+        assert(name);
```

Comment:
name is not used here, but there is an assert
On the other hand in automount_context_build_json name is also unused, but no assert. 
Not sure which is correct, but looks inconcistent